sanitizers/src/test/java/com/example/BUILD.bazel

```diff
@@ -314,6 +314,24 @@ java_fuzz_target_test(
     ],
 )
 
+java_fuzz_target_test(
```

Comment:
Should this have `tags = ["dangerous"]` in case it can trigger OS command execution?